src/anemoi/inference/testing/checks.py

```diff
@@ -78,6 +78,52 @@ def check_grib(
     assert all(curr > prev for prev, curr in zip(averages, averages[1:])), f"{check_accum} is not accumulating"
 
 
+@testing_registry.register("check_grib_lam")
+def check_grib_lam(
+    *,
+    file: Path,
+    checkpoint: "Checkpoint",
+    reference_grib: str,
+    reference_datetime: str | None = None,
+    mask="lam_0",
+    **kwargs,
+):
```

Comment:
Nit: is the cutout operation specific to the stretched grid model approach? If so, it could better to rename this `check_grib_sgm` (Stretched Grid Model). If it's common to both LAM and SGM, then maybe rename this to `check_grib_cutout`? Same for the function below.

Reply:
Good point, yes these checks are shared between SGM and LAM cutout. I renamed them in 3d33ee0410c36d6501d750e2f8854cb3a29c870e